test/support/autoinst_profile_examples.rb

```diff
@@ -0,0 +1,35 @@
+# encoding: utf-8
+
+# Copyright (c) [2016] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "y2storage/autoinst_profile/section_with_attributes"
+
+RSpec.shared_examples "autoinst section" do
+  let(:parent) do
+    Y2Storage::AutoinstProfile::SectionWithAttributes.new
+  end
+
+  describe ".new_from_hashes" do
```

Comment:
so common method for all sections?

Reply:
in general I see it as problem when you want to create new section.

Reply:
What I want to test is that the `parent` is set always when using `new_from_hashes`. About the method itself, we have a `new_from_hashes` and `new_from_storage` to build sections from the profile or from the storage layer.

What is exactly the problem?

Reply:
my problem is documentation to see what is important and I have to do when I want to create new section.